keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -356,6 +407,53 @@ type LinuxContainerUser struct {
 // }
 ```
 
+#### NodeFeatures in NodeStatus which contains SupplementalGroupsPolicy field
+
+```golang
+type NodeStatus struct {
+	// Features describes the set of implemented features implemented by the CRI implementation.
+	// +featureGate=SupplementalGroupsPolicy
+	// +optional
+	Features *NodeFeatures
+
+	// The available runtime handlers.
+	// +featureGate=RecursiveReadOnlyMounts
+	// +optional
+  RuntimeHandlers []RuntimeHandlers
```

Comment:
nit: indent

Reply:
Thanks for the catch! fixed in https://github.com/kubernetes/enhancements/pull/4728/commits/668d4a2eb8078a083adb69cda98c7fc5baf100ce